### DISCUSSION TITLE
Help to identify signal

### DISCUSSION THREAD

**Discussion by 8um1c4** [merbanan/rtl_433#2513] [General]
I receive a lot of those decoded signals:

     10 [00] {54} b5 7f 17 fb 7b b4 40 
      5 [00] {54} b5 7f 27 fb 78 fb 0c 
      5 [00] {54} b5 7f 27 fb 78 0f 34 
      2 [00] {54} b5 7f 27 fb 7e 81 04 
     10 [00] {54} b5 7f 47 fb 7e 5a 10 
      5 [00] {54} b5 7f 47 fb 7a d6 64 
      5 [00] {54} b5 7f 47 fb 79 38 2c 
      3 [00] {54} b5 7f 47 fb 7d b4 c8 
     20 [00] {54} b5 7f 27 fb 7c e9 8c 
     11 [00] {54} b5 7f 27 fb 7a 15 58 
      9 [00] {54} b5 7f 17 fb 7a 9a e4 
      3 [00] {54} b5 7f 47 fb 7b 23 fc 
      1 [00] {54} b5 7f 47 fb 7f af fc 
      5 [00] {54} b5 7f 47 fb 7c e1 e4 
      5 [00] {54} b5 7f 47 fb 7c 1d 18 
      5 [00] {54} b5 7f 47 fb 7e 5e e8 
      5 [00] {54} b5 7f 47 fb 7d d0 84 
      5 [00] {54} b5 7f 17 fb 78 6b b8 
      5 [00] {54} b5 7f 27 fb 78 e7 a4 
     10 [00] {54} b5 7f 27 fb 7a 19 a8 
      5 [00] {54} b5 7f 17 fb 7a 95 74 
      4 [00] {54} b5 7f 17 fb 7d 5a d0 
      3 [00] {54} b5 7f 27 fb 7d d6 b8 
      3 [00] {54} b5 7f 27 fb 7b 50 98 
      3 [00] {54} b5 7f 17 fb 7e f1 50 
      3 [00] {54} b5 7f 27 fb 7f 4c a8 
      5 [00] {54} b5 7f 17 fb 78 23 e8 
      5 [00] {54} b5 7f 27 fb 7c af e8 
      1 [00] {54} b5 7f 27 fb 7a 61 f8 
      5 [00] {54} b5 7f 27 fb 7e ed 08 
      5 [00] {54} b5 7f 17 fb 7d 12 a4 
      5 [00] {54} b5 7f 27 fb 7d 1e 08 

I don't know the origin, but it seems to be a garage or gate.

Some conclusions:
- The 10th digit (count from the right of course) is always 1, 2 or 4. Could bt command ID or button ID.
- Only the last 5 digits have different values
- The 5th digit (count from right) has only values bigger than 9, so the first 2 bits are always 1 (11xx)
- The last digit has  always values 0, 4 or 8, so last 2 bits are always 00.
- The number of bits with change are 16, so it could be a rolling code with 16 bits.
                     11xx xxxx xxxx xxxx xx00

Is this familiar for someone?
After find the origin it could be included in decoded commands?

Thanks.

**Comment by 8um1c4** [comment]
One of the signals decoded her:

https://triq.org/pdv/#AAB10402E0016C044C2BC48291A29291A291A291A292929292929291A291A1A1A29292929292929291A29291A292929291A29291A1A291A1A1A2929292929292929355

In rtl_433 appears: {54} b5 7f 47 fb 7b 23 fc
but in triq.ord appears {55} 5A BF 93 FD BE 8F 04

**Comment by zuckschwerdt** [comment]
You could try [a BitBench like this](https://triq.net/bitbench#c=%7B54%7Db57f17fb7bb440%5B10%5D&c=%7B54%7Db57f27fb78fb0c%5B%205%5D&c=%7B54%7Db57f27fb780f34%5B%205%5D&c=%7B54%7Db57f27fb7e8104%5B%202%5D&c=%7B54%7Db57f47fb7e5a10%5B10%5D&c=%7B54%7Db57f47fb7ad664%5B%205%5D&c=%7B54%7Db57f47fb79382c%5B%205%5D&c=%7B54%7Db57f47fb7db4c8%5B%203%5D&c=%7B54%7Db57f27fb7ce98c%5B20%5D&c=%7B54%7Db57f27fb7a1558%5B11%5D&c=%7B54%7Db57f17fb7a9ae4%5B%209%5D&c=%7B54%7Db57f47fb7b23fc%5B%203%5D&c=%7B54%7Db57f47fb7faffc%5B%201%5D&c=%7B54%7Db57f47fb7ce1e4%5B%205%5D&c=%7B54%7Db57f47fb7c1d18%5B%205%5D&c=%7B54%7Db57f47fb7e5ee8%5B%205%5D&c=%7B54%7Db57f47fb7dd084%5B%205%5D&c=%7B54%7Db57f17fb786bb8%5B%205%5D&c=%7B54%7Db57f27fb78e7a4%5B%205%5D&c=%7B54%7Db57f27fb7a19a8%5B10%5D&c=%7B54%7Db57f17fb7a9574%5B%205%5D&c=%7B54%7Db57f17fb7d5ad0%5B%204%5D&c=%7B54%7Db57f27fb7dd6b8%5B%203%5D&c=%7B54%7Db57f27fb7b5098%5B%203%5D&c=%7B54%7Db57f17fb7ef150%5B%203%5D&c=%7B54%7Db57f27fb7f4ca8%5B%203%5D&c=%7B54%7Db57f17fb7823e8%5B%205%5D&c=%7B54%7Db57f27fb7cafe8%5B%205%5D&c=%7B54%7Db57f27fb7a61f8%5B%201%5D&c=%7B54%7Db57f27fb7eed08%5B%205%5D&c=%7B54%7Db57f17fb7d12a4%5B%205%5D&c=%7B54%7Db57f27fb7d1e08%5B%205%5D&f=6h%208h8h%208h8h%208h8h&cw=4) to inspect the signal (also try just `v` as format).
If you look closely at the  pdv link you see it is indeed PWM 367/1100 but with preceeding gaps.
It looks like HCS200/HCS300 KeeLoq but we'd expect 66 bits.

**Reply by 8um1c4** [reply]
Yeap, thanks for the help.
Only 16 buts are changing, so it can't be Keelog. 
I don't know the source, so I have no way to test (trigger). 

By the way the number on the left of [00] is the number of occurrences in the last 2 or 3 days
